### ISSUE TITLE
Space/Shift toggle while moving

### ISSUE THREAD

**Issue by Ichisich** [RamonUnch/AltSnap#393]
Currently, MButton, X1 and X2 all toggle zone snapping by default, 
Giving X1 the shift property (inverting) and X2 the space property (disabling) would increase the utility out of the box.

**Comment by RamonUnch** [comment]
You are right, but actually those toggles should be settable as actions that would be reserved to the While moving and while resizing options.
I will first work on improving the Config dialog, not to list the same action list for all modes.
